scanners/pshtt.py

```diff
@@ -82,9 +82,10 @@ def scan(domain, options):
     yield row
 
 headers = [
-    "Canonical URL", "Live", "Redirect", "Redirect To",
+    "Domain", "Base Domain", "Canonical URL", "Live", "Redirect", "Redirect To",
```

Comment:
These two - `Domain` and `Base Domain` - should not be added to individual scanners. They are added for *all* reports by `domain-scan` automatically.